### ISSUE TITLE
Error occurs at def qat

### ISSUE THREAD

**Issue by momonoki3nenn** [Deci-AI/super-gradients#1104]
### 🐛 Describe the bug

Error messages generated

> Traceback (most recent call last):
  File "C:\Users\zynas\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    self._collect_stats(model, calib_data_loader, num_batches=num_calib_batches)
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\super_gradients\training\utils\quantization\calibrator.py", line 103, in _collect_stats
    model(local_image.to(device=device))
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\super_gradients\training\models\detection_models\customizable_detector.py", line 88, in forward
ll_impl
    return forward_call(*input, **kwargs)
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\super_gradients\training\models\detection_models\customizable_detector.py", line 88, in forward
    x = self.neck(x)
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\super_gradients\training\models\detection_models\yolo_nas\panneck.py", line 59, in forward
    x_n1_inter, x = self.neck1([c5, c4, c3])
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\super_gradients\training\models\detection_models\yolo_nas\yolo_stages.py", line 269, in forward
    x = self.upsample(x_inter)
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "c:\projects\evaluate_yolo_nas\venv\lib\site-packages\pytorch_quantization\nn\modules\quant_conv.py", line 342, in forward
    output_padding = self._output_padding(input, output_size, self.stride, self.padding, self.kernel_size) 
TypeError: _output_padding() missing 1 required positional argument: 'num_spatial_dims'

Here is my source code.

```
from pathlib import Path
from super_gradients.training import models, Trainer
from super_gradients.training.utils.quantization.selective_quantization_utils import SelectiveQuantizer
from super_gradients.training.utils.quantization.calibrator import QuantizationCalibrator
from super_gradients.training.utils.detection_utils import DetectionCollateFN
from super_gradients.training.datasets.datasets_utils import worker_init_reset_seed
from super_gradients.training.dataloaders.dataloaders import coco_detection_yolo_format_train, coco_detection_yolo_format_val
from super_gradients.training.losses import PPYoloELoss
from super_gradients.training.metrics import DetectionMetrics_050
from super_gradients.training.models.detection_models.pp_yolo_e import PPYoloEPostPredictionCallback

from multiprocessing import freeze_support

def train_int8(train_params:dict):

    trainer = Trainer(experiment_name = 'drone100_int8', ckpt_root_dir = 'checkpoints')
    train_data = coco_detection_yolo_format_train(
    dataset_params = {
        'data_dir': school['data_dir'],
        'images_dir': school['train_images_dir'],
        'labels_dir': school['train_labels_dir'],
        'classes': school['classes']
    },
    dataloader_params = {
        'batch_size':  8,
        'num_workers': 8,
        'shuffle': True,
        'pin_memory': True,
        'collate_fn':DetectionCollateFN(),
        'worker_init_fn': worker_init_reset_seed,
        'min_samples': 512
    }
)

    val_data = coco_detection_yolo_format_val(
        dataset_params = {
            'data_dir': school['data_dir'],
            'images_dir': school['val_images_dir'],
            'labels_dir': school['val_labels_dir'],
            'classes': school['classes']
        },
        dataloader_params = {
            'batch_size': 8,
            'num_workers': 8,
            'pin_memory': True,
            'collate_fn':DetectionCollateFN(),
            'worker_init_fn': worker_init_reset_seed
        }
    )
    model = models.get(
        'yolo_nas_s',
        num_classes = len(school['classes']),
        checkpoint_path = model_path.as_posix()
    )

    trainer.qat(
        model = model,
        training_params = train_params,
        train_loader = train_data,
        valid_loader = val_data,
        calib_loader = train_data
    )

if __name__ == '__main__':
    freeze_support()
    model_path = Path('checkpoints/drone100/ckpt_best.pth')
    
    train_params = train_params
    train_params['loss'] = PPYoloELoss(
        use_static_assigner=False,
        num_classes=len(school['classes']),
        reg_max=16
    )
    train_params['valid_metrics_list'] = [
        DetectionMetrics_050(
            score_thres=0.1,
            top_k_predictions=300,
            num_cls=len(school['classes']),
            normalize_targets=True,
            post_prediction_callback = PPYoloEPostPredictionCallback(
                score_threshold=0.01,
                nms_top_k=1000,
                max_predictions=300,
                nms_threshold=0.7
            )
        )
    ]
    train_params['max_epochs'] = 100

    train_int8(train_params)
```

Reference Site：https://colab.research.google.com/drive/1yHrHkUR1X2u2FjjvNMfUbSXTkUul6o1P?usp=sharing#scrollTo=cU9U1IaEerqX

Could train_loader, valid_loader, or calib_loader be the cause?

### Versions

OS : Windows11
Python : 3.9.13
super-gradients==3.1.1+master
torch == 1.13.1+cu117
torchaudio ==  0.13.1+cu117
torchvision ==  0.14.1+cu117

How to Install
pip install git+https://github.com/Deci-AI/super-gradients.git@master  --upgrade

**Comment by BloodAxe** [comment]
Yes, this is indeed a bug in torch quantization package when using newer versions of pytorch. 
We will fix that in 3.2 release

**Comment by shaydeci** [comment]
Im colsing this down as we already introduced a fix in the newer SG versions @momonoki3nenn .
Please re-open in case problem persists.
